checkstyle/suppressions.xml

```diff
@@ -10,7 +10,9 @@
 
     <!-- Clients -->
     <suppress checks="ClassFanOutComplexity"
-              files="(Fetcher|Sender|SenderTest|ConsumerCoordinator|KafkaConsumer|KafkaProducer|SaslServerAuthenticator|Utils|TransactionManagerTest|KafkaAdminClient|NetworkClient).java"/>
+              files="(Fetcher|Sender|SenderTest|ConsumerCoordinator|KafkaConsumer|KafkaProducer|Utils|TransactionManagerTest|KafkaAdminClient|NetworkClient).java"/>
+    <suppress checks="ClassFanOutComplexity"
+              files="SaslServerAuthenticator|SaslAuthenticatorTest"/>
```

Comment:
Should we add .java as other lines?

Reply:
We should double-check if `java` is needed. If not, then we should remove it from all files.

Reply:
@ijuma Checkstyle examples show .java suffix. So updated this one alone with the added suffix.